vsts/generate_javadocs_branch.ps1

```diff
@@ -56,85 +63,101 @@ function CreateJavadocReleaseBranch(
+    if ($UpdateProvisioningDeviceClientDocs -eq "True")
+    {
+        Remove-Item ..\$FolderName\provisioning\provisioning-device-client -Force -Recurse 
+        New-Item -Path ..\$FolderName\provisioning\provisioning-device-client -ItemType Directory   
+        Copy-Item -Force -Path .\provisioning\provisioning-device-client\* -Destination ..\$FolderName\provisioning\provisioning-device-client
+        Copy-Item -Recurse -Force -Path .\provisioning\provisioning-device-client\com -Destination ..\$FolderName\provisioning\provisioning-device-client   
+    }
 
-    Copy-Item -Force -Path .\provisioning\security\x509-provider\* -Destination ..\provisioning\security\x509-provider
-    Copy-Item -Recurse -Force -Path .\provisioning\security\x509-provider\com -Destination ..\provisioning\security\x509-provider
+    if ($UpdateProvisioningServiceClientDocs -eq "True")
+    {
+        Remove-Item ..\$FolderName\provisioning\provisioning-service-client -Force -Recurse
+        New-Item -Path ..\$FolderName\provisioning\provisioning-service-client -ItemType Directory
+        Copy-Item -Force -Path .\provisioning\provisioning-service-client\* -Destination ..\$FolderName\provisioning\provisioning-service-client
+        Copy-Item -Recurse -Force -Path .\provisioning\provisioning-service-client\com -Destination ..\$FolderName\provisioning\provisioning-service-client
+    }
 
-    # Create the folder to place content in if it does not exist.
-    if (Test-Path ..\$FolderName)
+    if ($UpdateTpmProviderEmulatorDocs -eq "True")
     {
-        Remove-Item ../$FolderName -Recurse
+        Remove-Item ..\$FolderName\provisioning\security\tpm-provider-emulator -Force -Recurse 
+        New-Item -Path ..\$FolderName\provisioning\security\tpm-provider-emulator -ItemType Directory
+        Copy-Item -Force -Path .\provisioning\security\tpm-provider-emulator\* -Destination ..\$FolderName\provisioning\security\tpm-provider-emulator
+        Copy-Item -Recurse -Force -Path .\provisioning\security\tpm-provider-emulator\com -Destination ..\$FolderName\provisioning\security\tpm-provider-emulator
     }
 
-    # Move the generated content to the correct folder. The folder will be different for master and preview branches.
-    New-Item -Path ../$FolderName -ItemType Directory
+    if ($UpdateTpmProviderDocs -eq "True")
+    {
+        Remove-Item ..\$FolderName\provisioning\security\tpm-provider -Force -Recurse
+        New-Item -Path ..\$FolderName\provisioning\security\tpm-provider -ItemType Directory
+        Copy-Item -Force -Path .\provisioning\security\tpm-provider\* -Destination ..\$FolderName\provisioning\security\tpm-provider
+        Copy-Item -Recurse -Force -Path .\provisioning\security\tpm-provider\com -Destination ..\$FolderName\provisioning\security\tpm-provider
+    }
 
-    if ($UpdateDepsDocs -eq "True")
+    if ($UpdateDiceProviderEmulatorDocs -eq "True")
```

Comment:
Dice?

Reply:
It is dice. I'm not sure what that package is though. We should ask Tim.